validation-service/src/main/java/magicthebuilder/validationservice/rabbit/CardReceiver.java

```diff
@@ -0,0 +1,27 @@
+package magicthebuilder.validationservice.rabbit;
+
+import magicthebuilder.cardservice.entity.MtgCard;
+import magicthebuilder.validationservice.repository.CardRepository;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+public class CardReceiver {
+
+    @Autowired
+    public void setCardRepository(CardRepository cardRepository) {
+        this.cardRepository = cardRepository;
+    }
+
+    private CardRepository cardRepository;
+
+    @RabbitListener(queues = "#{deleteQueue.name}")
+    public void receive1(List<MtgCard> newCards) {
+        var cards = cardRepository.getCards();
+        newCards.forEach(c -> cards.put(c.getId(), c));
```

Comment:
Czy to się wykonuje w tranzakcji i zmienia bazę? Nie powinno. Seriwsy powinny być w tranzakcji. Receiver mógłby delegować coś do serwisu.